src/berry_pots_app.c

```diff
@@ -141,7 +141,7 @@ extern const UnkStruct_0200D2B4 ov17_02203EF8[2];
 extern const UnkStruct_0200D2B4 ov17_02203F48;
 extern const UnkStruct_0200D2B4 ov17_02203F70;
 extern const UnkStruct_0200D2B4 ov17_02203F98;
-extern const u8 ov17_02203FC0[4];
+extern const TouchscreenHitbox ov17_02203FC0;
```

Comment:
A lot of these touchscreen hitbox struct renames can likely be documented now if you wanted to go through quickly and see if there's anything obvious

Reply:
Most of them haven't hit C yet so ruling this out-of-scope for now.